WordPress/wporg-main-2022
# About: Add the full About section menu to the About pages

Update the local navigation in the About section to be consistent across all About subpages, and add the menu to the main About page.

Requires https://github.com/WordPress/wporg-mu-plugins/pull/570. Fixes #385.

Note: This does not add any of the iterations proposed in https://github.com/WordPress/wporg-mu-plugins/issues/535, it only updates the navigation content.

~This change makes `_nav-about-details.php`, `_nav-about-people.php`, and `_nav-about-technology.php` have all the same content, but I've left them separate in case we decide to bring back this semi-sectioning idea. Happy to switch to a single `_nav-about-subpage.php` file instead if we'd rather be aggressively DRY.~ Update: Changed this to `_nav-about-subpage.php`, removed the other files, and updated all the page templates to call the new pattern.

### Screenshots

| Default | With dropdown open |
|---|---|
| ![about](https://github.com/WordPress/wporg-main-2022/assets/541093/8e2802e9-b358-49c3-a44a-1e9fd5e2ec49) | ![about-dropdown](https://github.com/WordPress/wporg-main-2022/assets/541093/05440c1c-4a2a-4202-a19b-21662b8e08fc) |
| ![about-subpage](https://github.com/WordPress/wporg-main-2022/assets/541093/f453d19b-0428-4a8d-b976-0ca593ec8e4b) | ![about-subpage-dd](https://github.com/WordPress/wporg-main-2022/assets/541093/1a280b4f-2ef3-4920-a2e7-24f765c9276d) |

And on small screens:

![about-mobile](https://github.com/WordPress/wporg-main-2022/assets/541093/a96c4ed0-0438-4d25-9fc9-4336f9a23906)

### How to test the changes in this Pull Request:

Use https://github.com/WordPress/wporg-mu-plugins/pull/570 for mu-plugins, otherwise there will be PHP errors.

1. View the About page
2. There should be a local navigation bar with 3 dropdowns
3. Click any page, the same navigation should appear on the subpages
4. Try the same with a small screen, the navigation should work as expected
5. View a Download subpage, there should be no change to the navigation here
6. View the footer of the About page, the 3 menus in the page content should still work
